### PR TITLE
Fork repo

### PR DESCRIPTION
**Pull request by jnevelson**
### Description

Thanks for contributing this Pull Request. Make sure that you send in this Pull Request to the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues :

- What Changed and Why? :

- Type of change :
  - [x] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [ ] Addition or Improvement of documentation

- Other:
  - [ ] Add unit tests
  - [ ] Add documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [1.5.0] - 2020-06-23
```
Comment (AgarFu) [inline CHANGELOG.md:7]: This is a pretty big change in behavior, shall we bump the major version?

File: gordian/repo.py [hunk 2]
```diff
@@ -12,28 +13,29 @@
 
 class Repo:
 
-    def __init__(self, repo_name, github_api_url=None, branch=None, git=None, files=None, semver_label=None, target_branch='master'):
+    def __init__(self, repo_name, github_api_url=None, branch=None, github=None, files=None, semver_label=None, target_branch='master'):
```
Comment (AgarFu) [inline gordian/repo.py:16]: This is a breaking change

File: gordian/repo.py [hunk 3]
```diff
@@ -104,15 +109,32 @@ def find_file(self, filename):
...
         try:
-            logger.debug(f'Creating branch {self.branch_name}')
-            ref = self._repo.create_git_ref(ref=self.branch_name, sha=sb.commit.sha)
+            ref = self._forked_repo.create_git_ref(ref=self.branch_name, sha=branch.commit.sha)
         except GithubException as e:
-            print(f"Branch {self.branch_name} already exists in github")
+            logger.debug(f'Branch {self.branch_name} already exists in github')
         self.branch_exists = True
 
+    def _get_branch(self, tries=1):
```
Comment (AgarFu) [inline gordian/repo.py:124]: would you consider to use https://pypi.org/project/retry/
```python
@retry(GithubException, tries=3, delay=1, backoff=2)
def _get_branch(self):
 ...
```

Comment (jnevelson) [reply gordian/repo.py:124]: Mind blown, did not know about this... 🤯

**Pull request merged**